include/fontgrp.h

```diff
@@ -228,11 +228,11 @@ void EndGreenText(void);
 // ??? GetSpecialCharChr(???);
 void PutSpecialChar(u16 * tm, int color, int id);
 // ??? PutNumberExt(???);
-void PutNumber(u16 *a, int b, int c);
-void PutNumberOrBlank(u16* a, int b, int c);
-void PutNumberTwoChr(u16 *a, int b, int c);
-void PutNumberSmall(u16* a, int b, int c);
-void PutNumberBonus(int a, u16 *b);
+void PutNumber(u16 * tm, int color, int number);
```

Comment:
What is `tm` here? From context, maybe something like `canvas` or `buffer` might be more appropriate?

Reply:
`tm` is generally used to indicate `tile map` (in the sense referring to the buffer that gets copied to the hardware background) -- @StanHash has a definition for it in the FE6 decomp's [`GLOSSARY.md`](https://github.com/FireEmblemUniverse/fireemblem6j/blob/main/docs/GLOSSARY.md#tmtilemap)

Reply:
hmm, if that's the convention used elsewhere then i guess it's fine. I'd much rather just write out `tilemap`.

Reply:
I'll keep this in mind for future cleanup/refactoring!